tests/test_mortgage_rate_crud.py

```diff
@@ -0,0 +1,125 @@
+        id = create_mortgage_rate.json['data']['id']
+        url = f'/api/v1/mortgage_rate/{id}'
+        # When
+        updated_payload = {
+            "credit_score_floor": "Testing",
+            "credit_score_cieling":"Testing",
+            "rate": 777.0
+            }
+
+        response = self.app.put(url, headers={"Content-Type": "application/json"}, data=json.dumps(updated_payload))
+        confirmation_url = response.json['data']['confirmation']['url']
+
+        # Then
+        confirmation = self.app.get(confirmation_url, headers={"Content-Type": "application/json"})
+        confirmation_body = confirmation.json['data']['attributes']
+        self.assertEqual(updated_payload['credit_score_floor'], confirmation_body['credit_score_floor'])
+        self.assertEqual(payload['credit_score_cieling'], confirmation_body['credit_score_cieling'])
+
+        self.assertNotEqual(payload['rate'], confirmation_body['rate'])
+
+        self.assertEqual(202, response.status_code)
+
+    # GET all
+    def test_succssful_get_mortgage_rate(self):
+        # Given
+
+        self.app.post('/api/v1/mortgage_rate', headers={"Content-Type": "application/json"}, data=json.dumps(range_620_639))
+        self.app.post('/api/v1/mortgage_rate', headers={"Content-Type": "application/json"}, data=json.dumps(range_640_659))
+        self.app.post('/api/v1/mortgage_rate', headers={"Content-Type": "application/json"}, data=json.dumps(range_660_679))
+        self.app.post('/api/v1/mortgage_rate', headers={"Content-Type": "application/json"}, data=json.dumps(range_680_699))
+        self.app.post('/api/v1/mortgage_rate', headers={"Content-Type": "application/json"}, data=json.dumps(range_700_759))
+        self.app.post('/api/v1/mortgage_rate', headers={"Content-Type": "application/json"}, data=json.dumps(range_760_850))
+
+        # When
+        response = self.app.get('/api/v1/mortgage_rate', headers={"Content-Type": "application/json"})
+        body = response.json['data']
+        returned_range_620_639 = body['range_620_639']
+        returned_range_640_659 = body['range_640_659']
+        returned_range_660_679 = body['range_660_679']
+        returned_range_680_699 = body['range_680_699']
+        returned_range_700_759 = body['range_700_759']
+        returned_range_760_850 = body['range_760_850']
+
+        # Then
+        self.assertEqual(200, response.status_code)
+        self.assertEqual(6, len(body))
+        self.assertEqual(range_620_639, returned_range_620_639['attributes'])
+        self.assertEqual(range_640_659, returned_range_640_659['attributes'])
+        self.assertEqual(range_660_679, returned_range_660_679['attributes'])
+        self.assertEqual(range_680_699, returned_range_680_699['attributes'])
+        self.assertEqual(range_700_759, returned_range_700_759['attributes'])
+        self.assertEqual(range_760_850, returned_range_760_850['attributes'])
+
+
+    # DESTROY
+    def test_successful_delete_mortgage_rate(self):
```

Comment:
When we explore why DELETE (and maybe POST/PUT) aren't returning the JSON response, and only returning the correct status code, we should ensure our tests are robust and catch that bug